app/src/main/java/org/p2p/wallet/settings/ui/newreset/pin/NewForgotPinBottomSheet.kt

```diff
@@ -0,0 +1,48 @@
+package org.p2p.wallet.settings.ui.newreset.pin
+
+import androidx.core.os.bundleOf
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.setFragmentResult
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.p2p.wallet.R
+import org.p2p.wallet.common.ui.NonDraggableBottomSheetDialogFragment
+import org.p2p.wallet.databinding.DialogNewForgotPasswordBinding
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+
+private const val EXTRA_REQUEST_KEY = "EXTRA_REQUEST_KEY"
+private const val EXTRA_RESULT_KEY = "EXTRA_RESULT_KEY"
+
+class NewForgotPinBottomSheet : NonDraggableBottomSheetDialogFragment() {
+
+    companion object {
+        fun show(fm: FragmentManager, requestKey: String, resultKey: String) =
+            NewForgotPinBottomSheet().withArgs(
+                EXTRA_REQUEST_KEY to requestKey,
+                EXTRA_RESULT_KEY to resultKey
+            ).show(fm, NewForgotPinBottomSheet::javaClass.name)
```

Comment:
```suggestion
            NewForgotPinBottomSheet()
                .withArgs(
                    EXTRA_REQUEST_KEY to requestKey,
                    EXTRA_RESULT_KEY to resultKey
                )
                .show(fm, NewForgotPinBottomSheet::javaClass.name)
```
every line break - a new sentence, a new intent like in Russian language

you create fragment
you add some args
you show it

not 
you create fragment add some args
       you show it

Reply:
yeah, agree. 
just copy pasted